ofrak_core/ofrak/ofrak_cli.py

```diff
@@ -12,6 +18,8 @@
 from ofrak.model.component_model import ComponentExternalTool
 from synthol.injector import DependencyInjector
 
+import ofrak.gui.server as server
```

Comment:
We don't typically import like this in OFRAK, I prefer the `from X import Y` style. There are a couple exceptions, like `logging`.